filebeat/tests/system/test_multiline.py

```diff
@@ -71,6 +71,38 @@ def test_c_style_log(self):
         # Check that output file has the same number of lines as the log file
         assert 4 == len(output)
 
+    def test_nginx_multiline_log(self):
+        """
+        Test an nginx multiline log
+        Special about this log file is that it has empty new lines
+        """
+        self.render_config_template(
+                path=os.path.abspath(self.working_dir) + "/log/*",
+                multiline=True,
+                pattern="^=[A-Z]+",
+                match="after",
+                negate="true",
```

Comment:
For example see this playground: http://play.golang.org/p/ABEJaX_lsK

Using the regex `^=[A-Z]+|^$'`, all empty lines become line events of length 0. These empty events should be quite easy filterable via exclude_lines